src/extensions/default/CodeSnippetsAdder/main.js

```diff
@@ -0,0 +1,34 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, regexp: true, indent: 4, maxerr: 50 */
+/*global define, brackets */
+
+define(function (require, exports, module) {
+    "use strict";
+
+    var CommandManager = brackets.getModule("command/CommandManager");
+	var EditorManager  = brackets.getModule("editor/EditorManager");
+
+	var CMD_ADD_CODE_SNIPPET_TEXT   = "Add Code Snippets";
+    var CMD_ADD_CODE_SNIPPET_ID     = "bramble.addCodeSnippet";
+    
+	function addCodeSnippet(snippet) {
+		var editor = EditorManager.getCurrentFullEditor();
+		if (editor) {
+			var insertionPos = editor.getCursorPos();
```

Comment:
@humphd Ok I've been doing some tests. And I noticed that when you haven't interacted with the editor as yet. When `getCursorPos()` is called it will default to **line 1** as you can see in my example below.

![framed_bramble_-_google_chrome_2017-02-08_12_45_25](https://cloud.githubusercontent.com/assets/10361408/22751224/a7d945ac-ee01-11e6-8b00-aeaaab195881.gif)